src/pybind/mgr/volumes/fs/operations/versions/metadata_manager.py

```diff
@@ -19,6 +19,16 @@
 
 log = logging.getLogger(__name__)
 
+
+def _conf_reader(fs, fd, offset=0, length=4096):
```

Comment:
Any specific reason to have `lenght=4096` ?
I can see existing `MAX_IO_BYTES = 8 * 1024`.

Reply:
4K is the page size of a typical system.